src/components/BulkMetadataForm/__stubs__/schema.ts

```diff
@@ -0,0 +1,34 @@
+import { CustomJsonSchema, MetadataProperties as fields } from '../../../types';
+
+export const schema: CustomJsonSchema = {
+	type: 'object',
+	properties: fields.reduce(
+		(
+			acc,
+			{ key, scope, type, format, 'bmi-isNotEmpty': isNotEmpty, 'bmi-errorMessage': customErrorMessage, label },
+		) => {
+			acc[key] = {
+				type: 'object',
+				properties: {
+					name: {
+						type: 'string',
+					},
+					value: {
+						type,
+						format,
+						'bmi-isNotEmpty': isNotEmpty,
+						errorMessage: {
+							format: customErrorMessage ?? "Het format voor '" + label + "' is ongeldig",
```

Comment:
Using a template literal here would make it more readable IMHO.

Reply:
Fixed